Turf.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "Turf"
-  s.version = "2.0.0-beta.1"
+  s.version = "2.0.0-rc1"
```

Comment:
`rc.1`?

Reply:
IMO it's important as `rc1` and `rc.1` are not the same in terms of semver

Reply:
Yes, we've [discussed](https://mapbox.slack.com/archives/CU0LHNNJY/p1628065685013700) that it is OK to go for RC for Turf since there are no more changes planned.

Reply:
Oh, now I see there's a misstype!